src/Google.Cloud.Language.V1Beta1/project.json

```diff
@@ -0,0 +1,39 @@
+﻿{
+  "version": "0.1.0-*",
+  "summary": "Google Cloud Language v1beta1 client library",
+  "description": "Google Cloud Language v1beta1 client library",
```

Comment:
I think these should be "Google Cloud Natural Language API" as per https://cloud.google.com/natural-language/docs/